_topics/en/ephemeral-anchors.md

```diff
@@ -19,8 +19,11 @@ topic-categories:
 ## Optional.  Produces a Markdown link with either "[title][]" or
 ## "[title](link)"
 primary_sources:
+    - title: BIP432
```

Comment:
Link broken and PR to add 432 is closed

Reply:
@jonatack do you have another suggestion for a source here? Otherwise we can drop this source for now and get this merged.